.github/workflows/release.yml

```diff
@@ -13,12 +13,39 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Build and push image
-        uses: docker/build-push-action@v1
+      - name: Get Version
+        id: info
+        run: |
+          VERSION=$(shell ./scripts/git-version)
+          echo ::set-output name=version::${VERSION}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        with:
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+          version: latest
+          driver-opts: image=moby/buildkit:master
```

Comment:
If we add a TODO to the ci, we should do so here as well

Reply:
Added